dv/uvm/core_ibex/scripts/compile-tb.py

```diff
@@ -35,6 +36,41 @@ def main() -> int:
 
     os.makedirs(args.output, exist_ok=True)
 
+    # Find the correct flags for the tb to link against the compiled ISS
+    cosim_subst_vars = {}
+    spike_iss_pc = ['riscv-riscv', 'riscv-disasm', 'riscv-fdt']
+    iss_pkgconfig_dict = {
+        'ISS_CFLAGS'  : ['--cflags'],
+        'ISS_LDFLAGS' : ['--libs-only-other'],
+        'ISS_LIBS'    : ['--libs-only-l', '--libs-only-L'],
+    }
+    if args.en_cosim:
+
+        try:
+            subprocess.check_output(['pkg-config'] + ['--exists'] + spike_iss_pc)
+        except subprocess.CalledProcessError as err:
+            raise RuntimeError(
+                'Failed to find ' + str(spike_iss_pc) + ' pkg-config packages. '
+                'Did you set the PKG_CONFIG_PATH correctly?') from err
```

Comment:
This is easier with format strings:
```
                f'Failed to find {spike_iss_pc} pkg-config packages. '
                f'Did you set the PKG_CONFIG_PATH correctly?') from err
```
(the second `f'...'` is optional, but I tend to do it to make everything line up!)

You also don't need the `from err` bit: that's the default behaviour.